src/redis_cache.nim

```diff
@@ -119,17 +119,17 @@ proc getCachedPhotoRail*(name: string): Future[PhotoRail] {.async.} =
     result = await getPhotoRail(name)
     await cache(result, name)
 
-proc getCachedList*(username=""; name=""; id=""): Future[List] {.async.} =
-  let list = if id.len > 0: redisNil
-             else: await get(toLower("l:" & username & '/' & name))
+proc getCachedList*(username=""; slug=""; id=""): Future[List] {.async.} =
+  let list = if id.len == 0: redisNil
+             else: await get("l:" & id )
 
   if list != redisNil:
     result = fromFlatty(uncompress(list), List)
   else:
     if id.len > 0:
       result = await getGraphListById(id)
     else:
-      result = await getGraphList(username, name)
+      result = await getGraphList(username, slug)
```

Comment:
Since `slug` is now only for redirecting, these functions could be renamed to `getGraphList` and `getGraphListBySlug`